frontend/src/layout/header/index.tsx

```diff
@@ -21,8 +23,19 @@ type Props = {
 
 export const Header: React.FC<Props> = ({ hideNavIcon = false, loginMode = false }) => {
     const menu = useMenu();
+    const router = useRouter();
+    const q = screenWidthAtMost(NAV_BREAKPOINT).replace(/^@media /, "");
+    const onNarrowScreen = window.matchMedia(q).matches;
```

Comment:
That's.. interesting. Not sure if I'd prefer just ``window.matchMedia(`(max-width: ${NAV_BEAKPOINT})`).matches``.

But I guess calling `screenWidthAtMost` is also fine.